examples/logrotate.cron

```diff
@@ -1,8 +1,23 @@
-#!/bin/sh
+#!/bin/bash
 
-/usr/sbin/logrotate /etc/logrotate.conf
-EXITVALUE=$?
-if [ $EXITVALUE != 0 ]; then
-    /usr/bin/logger -t logrotate "ALERT exited abnormally with [$EXITVALUE]"
+# exit immediately if there is another instance running
+if pgrep /usr/sbin/logrotate > /dev/null ; then
+	/bin/logger -p cron.warning -t logrotate "ALERT another instance of logrotate is running - exiting"
+	exit 1
 fi
```

Comment:
In my opinion, it would be safer to store logrotate's PID to a file and compare both PID and process name on the subsequent run (in case the PID file still exists).  Not sure if we can portably implement this without extending logrotate to optionally create the PID file...